de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/user/UserRepository.java

```diff
@@ -40,9 +39,14 @@
 @Repository
 public class UserRepository
 {
-	public User getById(@NonNull final UserId userId)
+	public User getByIdInTrx(@NonNull final UserId userId)
```

Comment:
If the parameter of the method that I invoke is `@NonNull`, I as a dev expect my returnvalue to be not-`null` as well.
Therefore, please see if we *really* need to return null - maybe we can also `Check.assumeNotNull` - and if so, rename the method to `getByIdInTrxOrNull`

Reply:
We don't need it to return null.
Done